src/sourabh/CodingExam_1/Sum.java

```diff
@@ -0,0 +1,36 @@
+/*Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+*/
+
+package sourabh.CodingExam_1;
+
+public class Sum {
+	int[] getSum(int[] num,int targetSum) {
+		
+		for(int index=0; index<num.length; index++) {
+			for(int index1=0; index1<num.length; index1++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files